SmartDeviceLink/SDLStreamingAudioLifecycleManager.m

```diff
@@ -195,6 +196,20 @@ - (void)sdl_handleAudioStartServiceAck:(SDLProtocolMessage *)audioStartServiceAc
     [self.audioStreamStateMachine transitionToState:SDLAudioStreamManagerStateReady];
 }
 
+- (void)handleProtocolEndService:(SDLProtocolMessage *)endService forSession:(Byte)session {
```

Comment:
Check here if the service type is video. Otherwise we will respond to both video and audio end session control frames.  

```
if (endService.header.serviceType != SDLServiceTypeAudio) { return; }
```

Reply:
Fixed in most recent commit.